packages/stream_video/lib/src/call/state/mixins/state_coordinator_mixin.dart

```diff
@@ -250,8 +254,59 @@ mixin StateCoordinatorMixin on StateNotifier<CallState> {
     );
   }
 
+  void coordinatorCallClosedCaptionsStarted(
+    StreamCallClosedCaptionsStartedEvent event,
+  ) {
+    final status = state.status;
+    if (status is! CallStatusActive) {
+      _logger.w(
+        () =>
+            '[coordinatorCallClosedCaptionsStarted] rejected (status is not Active)',
+      );
+      return;
+    }
+
+    state = state.copyWith(
+      isCaptioning: true,
+    );
+  }
+
+  void coordinatorCallClosedCaptionsStopped(
+    StreamCallClosedCaptionsStoppedEvent event,
+  ) {
+    final status = state.status;
+    if (status is! CallStatusActive) {
+      _logger.w(
+        () =>
+            '[coordinatorCallClosedCaptionsStopped] rejected (status is not Active)',
+      );
+      return;
+    }
+
+    state = state.copyWith(
+      isCaptioning: false,
+    );
+  }
+
+  void coordinatorCallClosedCaptionsFailed(
+    StreamCallClosedCaptionsFailedEvent event,
+  ) {
+    final status = state.status;
+    if (status is! CallStatusActive) {
+      _logger.w(
+        () =>
+            '[coordinatorCallClosedCaptionsFailed] rejected (status is not Active)',
+      );
+      return;
+    }
+
+    state = state.copyWith(
+      isTranscribing: false,
+    );
```

Comment:
I don't understand how closed captions failing affects transcription here?

Reply:
oh, good catch! Copy-paste mistake fixed that